Makefile

```diff
@@ -1,4 +1,4 @@
-GOVUK_ROOT_DIR="${HOME}/govuk"
+GOVUK_ROOT_DIR ?= "${GOVUK_APPS_HOME}/govuk"
```

Comment:
`GOVUK_APPS_HOME` is no longer being set, that should be `HOME`

Reply:
Good spot, updated now